src/boot/efi/log.h

```diff
@@ -21,11 +22,13 @@ __attribute__((no_stack_protector, noinline)) void __stack_chk_guard_init(void);
 
 _noreturn_ void freeze(void);
 void log_wait(void);
-_gnu_printf_(2, 3) EFI_STATUS log_internal(EFI_STATUS status, const char *format, ...);
-#define log_error_status(status, ...) log_internal(status, __VA_ARGS__)
-#define log_error(...) log_internal(EFI_INVALID_PARAMETER, __VA_ARGS__)
-#define log_oom() log_internal(EFI_OUT_OF_RESOURCES, "Out of memory.")
-#define log_trace() log_internal(EFI_SUCCESS, "%s:%i@%s", __FILE__, __LINE__, __func__)
+_gnu_printf_(3, 4) EFI_STATUS log_internal(EFI_STATUS status, uint8_t text_color, const char *format, ...);
+#define log_debug(...) log_internal(EFI_SUCCESS, EFI_MAGENTA, __VA_ARGS__)
+#define log_info(...) log_internal(EFI_SUCCESS, EFI_GREEN, __VA_ARGS__)
+#define log_error_status(status, ...) log_internal(status, EFI_LIGHTRED, __VA_ARGS__)
+#define log_error(...) log_internal(EFI_INVALID_PARAMETER, EFI_LIGHTRED, __VA_ARGS__)
+#define log_oom() log_internal(EFI_OUT_OF_RESOURCES, EFI_LIGHTRED, "Out of memory.")
+#define log_trace() log_internal(EFI_SUCCESS, EFI_LIGHTRED, "%s:%i@%s", __FILE__, __LINE__, __func__)
```

Comment:
So, I'd be fine to extend this with more log levels, matching those we use in userspace, but I'd also prefer then if we follow the same color scheme, i.e. make info level just plain white, and debug level grey.